openfisca_core/scripts/__init__.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+import sys
+import importlib
+
+
+TAX_BENEFIT_SYSTEM_OPTIONS = {
+    'country_package': {
+        'short': 'c',
+        'help': 'country package to use. If not provided, an automatic detection will be attempted by scanning the python packages installed in your environment which name contains the word "openfisca".'
+        },
+    'extensions': {
+        'short': 'e',
+        'help': 'extensions to load',
+        'nargs': '*'
+        },
+    'reforms': {
+        'short': 'r',
+        'help': 'reforms to apply to the country package',
+        'nargs': '*'
+        }
+    }
```

Comment:
Relying on such a `dict` seems too much code factorization, I'd prefer having many `parser.add_argument` like in previous version before this patch, but factorized in `add_tax_benefit_system_arguments`.